omelet-core/src/main/java/omelet/data/DriverConfigurations.java

```diff
@@ -86,7 +86,7 @@ public String get() {
 				+ "/src/main/resources/chromedriver".replace("/",
 						File.separator)), phantomdriverpath(System
 								.getProperty("user.dir")
-								+ "/src/main/resources/phantomjs.exe".replace("/",
+								+ "/src/main/resources/phantomjs".replace("/",
```

Comment:
Thanks for reviewing the code Kapil,
I have added code and also tested this on both Windows and Linux machines, its working good.

Regarding testcases failure, can you provide some more info what and where to fix, so that i can do that.